src/renderer/modules/webpack/common/modal.ts

```diff
@@ -52,32 +52,32 @@ export interface ModalComponents {
   ModalCloseButton: ReactComponent<unknown>;
 }
 
-export type Modal = ModuleExports & {
+export interface Modal {
   openModal: (
     render: (props: ModalProps) => React.ReactNode,
     options?: ModalOptions,
     contextKey?: string,
   ) => string;
   closeModal: (modalKey: string, contextKey?: string) => void;
-  ModalSize: ModalSize;
-  Direction: string;
-  Align: string;
-  Justify: string;
-  Wrap: string;
-};
+  ModalSize: typeof ModalSize;
+  Direction: Record<string, string>;
+  Align: Record<string, string>;
+  Justify: Record<string, string>;
+  Wrap: Record<string, string>;
```

Comment:
Can we be more specific with the keys here

Reply:
Those are the key names within discord and throughout client modding lol. They give the classes for the modal.

Reply:
@12944qwerty but what are the keys?

Reply:
Ohhh, you want to limit the keynames... gotcha.